<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRasterTransparencyWidget</class>
 <widget class="QWidget" name="QgsRasterTransparencyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QgsCollapsibleGroupBox" name="gboxNoDataValue">
     <property name="title">
      <string>No data value</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">rastertransp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mSrcNoDataValueCheckBox">
          <property name="toolTip">
           <string>Use original source no data value.</string>
          </property>
          <property name="text">
           <string>No data value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSrcNoDataValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Original data source no data value, if exists.</string>
          </property>
          <property name="text">
           <string>&lt;src no data value&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblUserNoDataValueLabel">
          <property name="toolTip">
           <string>Additional user defined no data value.</string>
          </property>
          <property name="text">
           <string>Additional no data value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leNoDataValue">
          <property name="toolTip">
           <string>Additional user defined no data value.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QgsCollapsibleGroupBox" name="gboxGlobalTransp">
     <property name="title">
      <string>Global transparency</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">rastertransp</string>
     </property>
     <layout class="QGridLayout" name="_3">
      <item row="1" column="4">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>&lt;p align=&quot;right&quot;&gt;Full&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QSlider" name="sliderTransparency">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblTransparencyPercent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> 00%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QgsCollapsibleGroupBox" name="gboxCustomTransparency">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Custom transparency options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">rastertransp</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblTransparent">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Transparency band</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboxTransparencyBand">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Transparent pixel list</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTableWidget" name="tableTransparency">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="pbnExportTransparentPixelValues">
          <property name="toolTip">
           <string>Export to file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionFileSave.svg</normaloff>:/images/themes/default/mActionFileSave.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="pbnImportTransparentPixelValues">
          <property name="toolTip">
           <string>Import from file</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionFileOpen.svg</normaloff>:/images/themes/default/mActionFileOpen.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnDefaultValues">
          <property name="toolTip">
           <string>Default values</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionOpenTable.svg</normaloff>:/images/themes/default/mActionOpenTable.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnRemoveSelectedRow">
          <property name="toolTip">
           <string>Remove selected row</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnAddValuesFromDisplay">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Add values from display</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionContextHelp.png</normaloff>:/images/themes/default/mActionContextHelp.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pbnAddValuesManually">
          <property name="toolTip">
           <string>Add values manually</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
